.env.development

```diff
@@ -1,5 +1,4 @@
-#for dev env tell webpack.config.dev.js to install service worker
-REACT_APP_SERVICE_WORKER=false
+REACT_APP_SERVICE_WORKER=true
```

Comment:
i'm not sure about this, i think this makes localhost development annoying

Reply:
on local development we're using `.env`